crates/ruff_linter/src/rules/flake8_implicit_str_concat/snapshots/ruff_linter__rules__flake8_implicit_str_concat__tests__ISC003_ISC.py.snap

```diff
@@ -1,34 +1,70 @@
 ---
 source: crates/ruff_linter/src/rules/flake8_implicit_str_concat/mod.rs
 ---
-ISC.py:9:3: ISC003 Explicitly concatenated string should be implicitly concatenated
+ISC.py:9:3: ISC003 [*] Explicitly concatenated string should be implicitly concatenated
    |
  8 |   _ = (
  9 | /   "abc" +
 10 | |   "def"
    | |_______^ ISC003
 11 |   )
    |
+   = help: Remove redundant '+' operator to implicitly concatenate
 
-ISC.py:14:3: ISC003 Explicitly concatenated string should be implicitly concatenated
+ℹ Safe fix
+6  6  |     "def"
+7  7  | 
+8  8  | _ = (
+9     |-  "abc" +
+10    |-  "def"
+   9  |+  "abc" "def"
```

Comment:
Given that this is the motivation for the rule:

> For string literals that wrap across multiple lines, implicit string concatenation within parentheses is preferred over explicit concatenation using the + operator, as the former is more readable.

I think it's important that the fix preserves the fact that each literal is on its own line (or it produces ISC001).